addOns/exportreport/src/main/java/org/zaproxy/zap/extension/exportreport/ExportReportAPI.java

```diff
@@ -276,6 +279,11 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
                                     "exportreport.message.console.info.pass.generate"));
                 }
 
+                int scanId = this.getParam(params, ACTION_PARAM_SCAN_ID, -1);
```

Comment:
IMO we should validate here if the ID does not exist, if the user is asking for a specific scan we should either return just that or inform it does not exist.